rotkehlchen/db/updates.py

```diff
@@ -10,20 +10,23 @@
 from rotkehlchen.constants.timing import DEFAULT_TIMEOUT_TUPLE
 from rotkehlchen.errors.misc import RemoteError
 from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.fval import FVal
 from rotkehlchen.globaldb.handler import GlobalDBHandler
 from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import SupportedBlockchain
 from rotkehlchen.utils.misc import ts_now
 from rotkehlchen.utils.network import query_file
 
 if TYPE_CHECKING:
-    from rotkehlchen.db.dbhandler import DBHandler
+    from rotkehlchen.db.dbhandler import DBCursor, DBHandler
     from rotkehlchen.user_messages import MessagesAggregator
 
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
 LAST_DATA_UPDATES_KEY: Final = 'last_data_updates_ts'
 SPAM_ASSETS_URL = 'https://raw.githubusercontent.com/rotki/data/{branch}/updates/spam_assets/assets_v{version}.json'  # noqa: E501
+RPC_NODES_URL = 'https://raw.githubusercontent.com/rotki/data/{branch}/updates/rpc_nodes/v{version}.json'  # noqa: E501
```

Comment:
Hey @prettyirrelevant @yabirgb, here I see a discrepancy in naming that should be addressed.

Why do spam assets have the prefix `assets_` for the filename? They are always going to be the same under the directory `spam_assets` no?

Also @prettyirrelevant why is there no PR in the data repo for the `rpc_nodes` stuff?

Reply:
> Also @prettyirrelevant why is there no PR in the data repo for the rpc_nodes stuff?

No reason, was only waiting till  when the PR almost gets merged.